fcrepo-client-impl/src/main/java/org/fcrepo/client/impl/FedoraRepositoryImpl.java

```diff
@@ -143,7 +153,7 @@ public FedoraDatastream createDatastream(final String path, final FedoraContent
             final int statusCode = status.getStatusCode();
 
             if (statusCode == SC_CREATED) {
-                return getDatastream(path);
+                return getDatastream(prependTransactionId(path));
```

Comment:
To clarify, `getDatastream(path)` invokes `prependTransactionId()` on the arg `path`:
https://github.com/fcrepo4-labs/fcrepo4-client/pull/31/files#diff-c56ffc3d56d2f95d81b3208a7d18e4f1R137

Therefore, `getDatastream(prependTransactionId(path))` seems to be redundantly calling `prependTransactionId()`. no?